src/index.js

```diff
@@ -56,9 +56,11 @@ export function app(state, actions, view, container) {
   }
 
   function resolveNode(node) {
-    return typeof node === "function"
-      ? resolveNode(node(globalState, wiredActions))
-      : node
+    return node === null
+      ? h(node, null)
```

Comment:
Following our discussion in this PR, this line should definitely not lead to h(null)

Reply:
If I remove that line and I `return null` into `ComponentWithStateAndActions` it will throw an error on the console because it can't get `nodeName` from undefined `node.nodeName`